lib/events.js

```diff
@@ -139,7 +139,12 @@ EventEmitter.prototype.emit = function emit(type) {
       domain.emit('error', er);
     } else if (er instanceof Error) {
       throw er; // Unhandled 'error' event
+    } else if (typeof er === 'string') {
+      // er is a string, wrap it in an Error
+      throw new Error(er);
     } else {
+      // At least give some kind of context to the user
+      console.error(er);
```

Comment:
I'm not so sure about this. Can't we just attach `er` to the thrown error object and/or concatenate the `er.message` to the thrown error message?